device/peripherals/ADXL375/ADXL375.h

```diff
@@ -0,0 +1,163 @@
+//
+// Created by skippynose on 12/17/22.
+//
+
+#ifndef LAUNCH_CORE_ADXL375_H
+#define LAUNCH_CORE_ADXL375_H
+
+
+#include <stdlib.h>
+#include <stdint.h>
+
+#include "sched/macros.h"
+#include "return.h"
+#include "device/I2CDevice.h"
+
+
+class ADXL375 {
+public:
+    ADXL375(I2CDevice& i2c) : m_i2c(i2c)
+        {}
+
+   RetType readX(int16_t* xAxis){
+        // TODO read the values of the MSB and LSB and pass it into the final variable
+        RetType ret;
```

Comment:
Just declare this on line 31. You would also probably want the resume before this variable initialization too.